src/main/java/utils/Utility.java

```diff
@@ -2,6 +2,19 @@
 
 public class Utility {
 
+    public static class Temperature{
+
+        public static double Celsius(double fahrenheit){
+            return 5.0/9.0*(fahrenheit-32);
+        }
+
+        public static double Fahrenheit(double celsius){
```

Comment:
use names for methods so that you can understand what they do just from the name, ad example:
Temperature.**toCelsius()** or Temperature.**celsiusToFarenheit()**

Reply:
see above